ml_test/train_emulator.jl

```diff
@@ -0,0 +1,155 @@
+# Get ML packages
+import MLJ
+import Flux
+import MLJModels
+import MLJFlux
+Standardizer = MLJ.@load Standardizer pkg = MLJModels
+NeuralNetworkRegressor = MLJ.@load NeuralNetworkRegressor pkg = MLJFlux
+
+# path to our ML testing folder
+import CloudMicrophysics as CM
+fpath = joinpath(pkgdir(CM), "ml_test")
+
+# We will be precise
+const FT = Float64
+
+# Container for the NN we are about to build
+struct NNBuilder <: MLJFlux.Builder
+    layer_sizes::Vector{Integer}
+    dropout::Vector{Float64}
+end
+
+# TODO - what does it do exactly?
```

Comment:
I agree with this TODO - this could probably be simplified.

Reply:
Agreed. I don't have the time to do it now, but at would be nice to try to get rid of some of those packages